ra/ra_test.go

```diff
@@ -1382,11 +1380,10 @@ func TestNewOrder(t *testing.T) {
 	test.AssertNotError(t, err, "ra.NewOrder failed")
 	test.AssertEquals(t, orderA.RegistrationID, int64(1))
 	test.AssertEquals(t, orderA.Expires.AsTime(), now.Add(ra.profiles.def().orderLifetime))
-	test.AssertEquals(t, len(orderA.DnsNames), 3)
 	test.AssertEquals(t, len(orderA.Identifiers), 3)
 	test.AssertEquals(t, orderA.CertificateProfileName, "test")
-	// We expect the order names to have been sorted, deduped, and lowercased
-	test.AssertDeepEquals(t, orderA.DnsNames, []string{"a.com", "b.com", "c.com"})
+	// We expect the order's identifiers to have been sorted, deduped, and
+	// lowercased
```

Comment:
We could make this consistent with https://github.com/letsencrypt/boulder/pull/8100#discussion_r2029320271

```suggestion
	// We expect the order's identifiers to have been sorted, deduplicated, and
	// lowercase.
```